### DISCUSSION TITLE
Add video capability - just released

### DISCUSSION THREAD

**Discussion by mchaker** [hlky/stable-diffusion-webui#761] [Feature suggestions]
# https://github.com/nateraw/stable-diffusion-videos

please add to gradio UI 🙏

**Comment by ooofest** [comment]
That's not the only place where video capability is being developed, e.g., this has been getting some attention:

https://replicate.com/deforum/deforum_stable_diffusion/versions/e22e77495f2fb83c34d5fae2ad8ab63c0a87b6b573b6208e1535b23b89ea66d6
https://github.com/deforum/stable-diffusion

**Reply by mchaker** [reply]
Excellent, I'll add that to my list of links at https://links.sd soon
